src/discord/discord.service.ts

```diff
@@ -47,17 +44,21 @@ export class DiscordService {
     if (username) {
       updatedDiscord.username = username;
     }
-    if (description) {
-      updatedDiscord.bio.description = description;
+    if (bio) {
+      updatedDiscord.bio = bio;
     }
-    if (twitter) {
-      updatedDiscord.bio.twitter = twitter;
+
+    if (socials && socials.discord) {
+      updatedDiscord.socials.discord = socials.discord;
+    }
+    if (socials && socials.twitter) {
+      updatedDiscord.socials.twitter = socials.twitter;
     }
-    if (linkedin) {
-      updatedDiscord.bio.linkedin = linkedin;
+    if (socials && socials.linkedin) {
+      updatedDiscord.socials.linkedin = socials.linkedin;
     }
-    if (github) {
-      updatedDiscord.bio.github = github;
+    if (socials && socials.github) {
```

Comment:
In the future it would be good to move these to a config file, so it is easily configurable